samples/nrf9160/gnss/prj.conf

```diff
@@ -12,11 +13,21 @@ CONFIG_NEWLIB_LIBC=y
 CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
 CONFIG_AT_CMD=y
 
-# To use SUPL client library for assisted GPS, enable the options below
-CONFIG_SUPL_CLIENT_LIB=n
-CONFIG_LTE_LINK_CONTROL=n
-CONFIG_LTE_NETWORK_MODE_LTE_M_GPS=n
+# GNSS sample
+#CONFIG_GNSS_SAMPLE_NMEA_ONLY=y
+#CONFIG_GNSS_SAMPLE_ANTENNA_EXTERNAL=y
+# Enable to use nRF Cloud A-GPS
+#CONFIG_GNSS_SAMPLE_ASSISTANCE_NRF_CLOUD=y
```

Comment:
I think we could leave out the out-commented configs and rather present them in the sample docs